src/BenchmarksDriver/Program.cs

```diff
@@ -1231,6 +1245,44 @@ public static int Main(string[] args)
             return 0;
         }
 
+        private static async Task<int> UploadFileAsync(string filename, AttachmentLocation location, ServerJob serverJob, Uri serverJobUri)
+        {
+            try
+            {
+                var outputFileSegments = filename.Split(';');
+                var attachmentFilename = outputFileSegments[0];
+
+                if (!File.Exists(attachmentFilename))
+                {
+                    Console.WriteLine($"Output File '{attachmentFilename}' could not be loaded.");
+                    return 8;
+                }
+
+                var destinationFilename = outputFileSegments.Length > 1
+                    ? outputFileSegments[1]
+                    : Path.GetFileName(attachmentFilename);
+
+                var requestContent = new MultipartFormDataContent();
+
+                var fileContent = attachmentFilename.StartsWith("http", StringComparison.OrdinalIgnoreCase)
```

Comment:
var fileContent = attachmentFilename.StartsWith("http", StringComparison.OrdinalIgnoreCase)
                    ? new StreamContent(await _httpClient.GetByteArrayAsync(attachmentFilename))
                    : new StreamContent(new FileStream(attachmentFilename,FileMode.Open,FileAccess.Read,FileShare.ReadWrite,1,FileOptions.Asynchronous | FileOptions.SequentialScan));

Reply:
Is `1` for the buffer size intentional? default is 4K

Reply:
`0` throws, `1` says don't use internal buffer

Reply:
Came up in https://github.com/aspnet/KestrelHttpServer/pull/2049#discussion_r165830218

![image](https://user-images.githubusercontent.com/1142958/36556630-c0a247ba-17fd-11e8-9691-02bfbc88bdad.png)